modules/experimental/pages/manoeuvrable.adoc

```diff
@@ -0,0 +1,14 @@
+= Manoeuvrable
+
+`Unit ability`
+
+These War Engine units are unusually manoeuvrable.
+
+---
+
+Where your unit has the {manoeuvrable} ability:
+
+* You may make any number of free turns when moving the unit.
```

Comment:
Not sure if you've already considered this, but: This will usually result in the same outcome as a statement to the effect of 'move in any direction, with whatever facing you like'. However, I think that there is at least one edge case:

As currently written, if you move a war engine such that it hits a dangerous terrain, and you fail the test, it has to stop dead, facing in its direction of travel at that point, doesn't it? (So for example, if 'backwards' motion is interrupted in this way, the war engine would actually stop with its rear to the enemy rather than its front.)

So, do you intend this?

- If yes, then I wonder if we should make this consequence more obvious.
- If no, then we should change the text to make it clear that we mean in effect 'move in any direction while the war engine maintains any facing you like'.

Reply:
I _think_ that's why it was phrased that way, in case it was ever given to a War Engine unit that didn't have 360º arcs on all its weapon mounts.

But I don't have a strong preference if you think the rule would work better in 'move in any direction, with whatever facing you like' form.

Reply:
I don't have strong feeling on the particular choice either. The 'move in any direction, with whatever facing you like' form is of course more straightforward in game terms — so all else being equal, I guess I'd lean towards that.

Some might argue against its permissiveness, but I can just imagine that otherwise either:

1. People will play it this permissive way anyway, because the strict interpretation doesn't occur to them or they don't care for it.
2. People will have heated debates about the strict interpretation and its application.
3. We have to write in a fiddly explanation for a fiddly situation that will be pretty unusual anyway.

I think I've just talked myself more into the permissive option... 😆

Reply:
👍👍🤣

Reply:
How about something like this?
```suggestion
* When you move this unit, you are not limited by the usual rules for forward and reverse movement of War Engine units.
Instead, you may move this unit in any direction, and turn it to face any direction, without any cost to the unit's movement allowance.
```